src/css/styles.scss

```diff
@@ -17,6 +17,13 @@
   left: 0;
   right: 0;
 
+  // serves 2 purposes:
+  // 1. prevent selecting text outside the component when double-clicking or
+  //    dragging inside it (e.g. on canvas)
+  // 2. prevent selecting UI, both from the inside, and from outside the
+  //    component (e.g. if you select text in a sidebar)
+  user-select: none;
```

Comment:
So with this, the selection will also be prevented in the npm package which was not till now 👍

Reply:
@dwelle does the SCSS preprocessor handle vendor prefixing of this? If not, still need `-webkit-user-select` fallback for several browsers